lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -44,6 +44,7 @@
 #include "llvm/Support/raw_ostream.h"
 #include "llvm/Support/SaveAndRestore.h"
 #include <algorithm>
+#include "GenericSignatureBuilderImpl.h"
```

Comment:
Nitpick: LLVM style says local headers go at the top.

Reply:
Ah, of course. Committing locally to fix at some point, but I'm not going to re-start CI for that right now.